gauge.go

```diff
@@ -28,6 +29,7 @@ func main() {
 	defer recoverPanic()
 	if err := cmd.Parse(); err != nil {
 		logger.Info(true, err.Error())
+		syscall.Exit(1)
```

Comment:
@joshua-jin We use os.Exit(1) in most places, to exit with a non zero exit code. Doing the same here would help.

Reply:
Ok, it's done.